credentials/utopia-birth-certificate-vc-v2/queries.json

```diff
@@ -0,0 +1,24 @@
+{
+  "default": {
+    "type": "QueryByExample",
+    "credentialQuery": [
+      {
+        "reason": "Please present your Utopia Birth Certificate Verifiable Credential(s) to complete the verification process.",
+        "example": {
+          "@context": [
```

Comment:
Does this need to match the credential?

Reply:
...one would think that's why one would use a query. Can you say more about why you asked that? 😕

Reply:
I'm not very familiar with how this query system works.  It looked out of sync, so asked about it, and hope it will be tested and adjusted as needed.